### PR TITLE
Feature/custom tokens

### PR DESCRIPTION
**Pull request by LiorAgnin**
## Proposed changes

- New UI for choosing token

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: dapp/src/components/issuance/CurrencyType.jsx [hunk 1]
```diff
@@ -1,32 +1,60 @@
...
+// ]
+
+const CommunityTypes = [
+  { label: 'Mintable/Burnable token', icon: MintableBurnable, value: 'mintableBurnable', tooltipText: `This allows community manager to mint or burn tokens and fully control the token supply. Those types of tokens are used for credit-like or reward systems that can be used by any organization.` },
+  { label: 'One time issued token', icon: OneTimeIssuer, value: 'basic', tooltipText: `Those types of tokens could be minted once and their supply is fixed. This allows to create assets which are digitally scarce and can be produced in limited supply. Usually to achieve a higher level of protection and guarantee no one can create more coins of this type can protect its value and decentralization.` }
+  // { label: 'Use existing token', icon: DefaultIcon, value: 'existingToken', tooltipText: `Any existing coin that is ERC-20 compatible could be integrated into your community. We currently support DAI and will be adding more stable-coins very soon. Any stable coin that is operating on the Ethereum network could be plugged into your community. ` }
```
Comment (leonprou) [inline dapp/src/components/issuance/CurrencyType.jsx]: can we delete this?

File: dapp/src/components/issuance/CurrencyType.jsx [hunk 2]
```diff
@@ -1,32 +1,60 @@
...
 ]
 
-const Option = (props) => {
+// const Option = (props) => {
```
Comment (leonprou) [inline dapp/src/components/issuance/CurrencyType.jsx]: and this?

File: dapp/src/components/dashboard/containers/MainDashboard/index.jsx [hunk 3]
```diff
@@ -57,11 +56,17 @@ class DashboardLayout extends Component {
       if (window && window.analytics && isAdmin) {
         const { analytics } = window
 
-        analytics.identify(`${accountAddress}`, {
-          role: 'admin'
-        })
-
         if (location.pathname.includes('/justCreated')) {
+          analytics.ready(() => {
```
Comment (leonprou) [inline dapp/src/components/dashboard/containers/MainDashboard/index.jsx]: Two questions:
- Maybe we can move this logic out of component? to redux-saga by example. So we don't need the /justCreated parameter.
- If I issued one community, used the bridg (bridgeWasUsed is true). Then created a new one, `bridgeWasUsed` would be false then?

Comment (LiorAgnin) [reply dapp/src/components/dashboard/containers/MainDashboard/index.jsx]: 1. It's good advice. I'll change it.
2. I thought so but it turns out it doesn't work that way for some reason. I am still investigating the issue

File: dapp/src/components/issuance/CurrencyType.jsx (hunk 1 above)
Comment (LiorAgnin) [reply dapp/src/components/issuance/CurrencyType.jsx]: I'll remove all src/components/issuance/ issuace folder.

**Pull request merged**